### ISSUE TITLE
Cannot use with aeson 1 (lts-8.5)

### ISSUE THREAD

**Issue by ejconlon** [yamadapc/hpack-convert#5]
I get this error when running `stack install hpack-convert` with resolver `lts-8.5`:

```
Error: While constructing the build plan, the following exceptions were encountered:

In the dependencies for hpack-convert-0.14.6:
    aeson-1.0.2.1 must match ==0.11.* (latest applicable is 0.11.3.0)
```

Thanks :)

**Comment by ghost** [comment]
I use the following sometimes to avoid the having to check in a vanilla stack.yaml file (and add stack.yaml to .gitignore), in case it is helpful for you...It's probably not much better than having the stack.yaml file.

```
stack.yaml : 
> stack init --force --resolver lts-8.13
```

**Comment by yamadapc** [comment]
Will be published shortly in `v0.14.7`
